src/scss/components/contact.scss

```diff
@@ -0,0 +1,43 @@
+.contactUs {
+  @extend .container;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
These properties are not necessary because the default behavior of block elements is to be stacked vertically